### PR TITLE
adding new Linux answers

### PR DESCRIPTION
**Pull request by ismferd**
Adding new answers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -597,10 +603,16 @@ The tasks are stored in a cron file.
...
 <summary>How to add a new user to the system without providing him the ability to log-in into the system?</summary><br><b>
+
+  * adduser user_name --shell=/bin/false --no-create-home
+
 </b></details>
 
 <details>
 <summary>On a system which uses systemd, how would display the logs?</summary>
+
+  * journactl
```
Comment (bregman-arie) [inline README.md]: journalctl*

**Review by bregman-arie — CHANGES_REQUESTED** [review]
Thank you for the change. 
Let's merge it after fixing the journalctl command.

File: README.md (hunk 1 above)
Comment (ismferd) [reply README.md]: Thanks @bregman-arie , Could you check it again?

**Comment by bregman-arie** [comment]
Thanks for fixing it. Merging

**Pull request merged**